### PR TITLE
add eurKey keymap

### PR DESCRIPTION
**Pull request by KraXen72**
eurKey keymap is a keymap for europeans. it is basically the us keymap with a bunch of altgr codes
https://eurkey.steffen.bruentjen.eu/start.html
i added display support for it.
please ignore how it also took the slovak keymap commits, even though they are already merged, i don't know what to do about it

### CODE REVIEW

**Comment by KraXen72** [comment]
it works, but it's kinda a mess
image
i'm not sure i like it
i wanted it to display the keys where i use altgr, but it barely helps:
image
this is supposed to be ß and ä but it's barely visible

**Comment by KraXen72** [comment]
if you like it, you can merge it i guess, but i don't really care if you don't merge it. it didn't take that long to make, i just copied and pasted the keycodes from the eurKey layout site. 
TLDR probably don't merge this

File: src/main/python/keymaps.py [hunk 1]
```diff
@@ -11,6 +11,7 @@
     ("Norwegian (QWERTY)", norwegian.keymap),
     ("Russian (ЙЦУКЕН)", russian.keymap),
     ("Slovak (QWERTY)", slovak.keymap),
+    ("eurKey (QWERTY)", eurkey.keymap),
```
Comment (xyzz) [inline src/main/python/keymaps.py:14]: Please keep the list alphabetically sorted and rename the layout to EurKEY so as to match the official naming capitalization.

**Review by xyzz — CHANGES_REQUESTED** [review]

**Comment by KraXen72** [comment]
okay will do that, 
how do you feel about me removing the obvious shift + altgr ones?
for example, it's obvious that if Q has æ on altgr, shift + altgr will be Æ 
this is the practise for all letters. for symbols, i will keep any that are different than just capital version of the normal one.

**Comment by xyzz** [comment]
I don't mind either way.
